OctoKit/User.swift

```diff
@@ -19,6 +19,31 @@ open class User: Codable {
     open var numberOfPublicRepos: Int?
     open var numberOfPublicGists: Int?
     open var numberOfPrivateRepos: Int?
+    open var nodeId: String?
```

Comment:
Could we also uppercase `Id`?

```suggestion
    open var nodeID: String?
```